rest-openehr/src/main/java/org/ehrbase/rest/openehr/audit/support/OpenEhrAuditMessageBuilder.java

```diff
@@ -52,8 +52,13 @@ protected OpenEhrAuditMessageBuilder(
 
         addSourceActiveParticipant(auditDataset);
         addDestinationActiveParticipant();
+        addAuditSource(auditContext, auditDataset);
+    }
 
-        delegate.setAuditSource(auditContext);
+    protected final T addAuditSource(AuditContext auditContext, OpenEhrAuditDataset auditDataset) {
```

Comment:
why not private static?

Reply:
because it's potentially can be overrided

Reply:
Also, this is n "add", but a "set"